source/reference/api/reverse.txt

```diff
@@ -28,7 +28,8 @@ For example:
   writes.
 
 In this scenario, you can use the ``reverse`` endpoint to sync writes
-from ``cluster1`` to ``cluster0``.
+from ``cluster1`` to ``cluster0``, including any writes that
+occurred after ``mongosync`` reached ``canWrite=true``. 
```

Comment:
could we indicate that `canWrite` is found from the /progress endpoint? and could we link the [docs](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/progress/#successful-response) too?

Reply:
Fixed